modules/utxo-lib/package.json

```diff
@@ -47,6 +47,7 @@
   ],
   "dependencies": {
     "@bitgo/blake2b": "^3.2.1",
+    "@brandonblack/musig": "^0.0.1-alpha.0",
```

Comment:
is it possible have this under our bitgo scope?

Reply:
Not sure if it will or not. It's an open source project developed by me, and which I expect to be the maintainer of even if I am no longer a BitGo employee.